pages/common/lab.md

```diff
@@ -0,0 +1,19 @@
+# lab
+
+> A CLI for Gitlab.
+
+- Create a merge request and specify remote and target branch:
+
+`lab mr create {{remote}} {{target_branch}}`
+
+- The `lab` CLI can call into `hub` commands. Create a PR of the current branch in origin/master:
```

Comment:
The first sentence of the description can be the sub-heading for the page. You can mention ```It can call into `hub` commands.```